TripHouse/index.html

```diff
@@ -255,16 +297,186 @@ <h3 class="best-dest-text">Island</h3>
+            </svg>
+            <div class="third-col-content">
+              <p class="text-review hotel name">Hanna Ivanova</p>
+              <div class="country-cont">
+                <svg class="country-review" width="20" height="19">
+                  <use href="../images/svg-sprite.svg#ukraine"></use>
+                </svg>
+                <p class="text-country">Ukraine</p>
+              </div>
+            </div>
+          </div>
+          <div class="review-text">
+            <p class="review">
+              Great location, really pleasant and clean rooms, but the thing
+              that makes this such a good place to stay are the staff. All of
+              the people are incredibly helpful and generous with their time and
+              advice.
+            </p>
+          </div>
+        </div>
+        <div class="first-col-reviews Netherlands">
+          <div class="first-img-cont">
+            <div class="guests-wrapper">
+              <p class="price-hotel">From 40 USD</p>
+              <img
+                class="guests-img"
+                src="../images/king-kong.svg"
+                alt="King Kong hostel"
+              />
+            </div>
+          </div>
+          <div class="second-col-reviews">
+            <div class="second-col-content">
+              <p class="text-review hotel">King Kong Hostel</p>
+              <p class="text-review mark">8,9</p>
+            </div>
+            <div class="second-col-2content">
+              <p class="text-review city">Rotterdam, Netherlands</p>
+              <p class="text-review amount-reviews">2016 reviews</p>
+            </div>
+          </div>
+          <div class="third-col-reviews">
+            <svg class="user-logo-guests" width="54" height="54">
+              <use href="../images/svg-sprite.svg#user-logo-guests"></use>
+            </svg>
+            <div class="third-col-content">
+              <p class="text-review hotel name">Adam Smith</p>
+              <div class="country-cont">
+                <svg class="country-review" width="20" height="19">
+                  <use href="../images/svg-sprite.svg#usa"></use>
+                </svg>
+                <p class="text-country">USA</p>
+              </div>
+            </div>
+          </div>
+          <div class="review-text1">
+            <p class="review">
+              I've been in this hostel for a month and here is the best hostel
+              of all. Excellent management of excellent and ethical staff and
+              bedrooms Bathrooms are always clean and excellent Most
+              importantly, those who work here are kind and good-natured.
+            </p>
+          </div>
+        </div>
+        <div class="first-col-reviews Marocco">
+          <div class="first-img-cont">
+            <div class="guests-wrapper">
+              <p class="price-hotel">From 100 USD</p>
+              <img
```

Comment:
We can divide graphic formats: vector(svg) and raster(jpeg, png)
If we use somethings that looks like icons, it's better to use svg
If we use photo or drawings, it's better to use jpeg or png
Moreover we have new modern formats of graphics, they are really cool - [read more](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types)


Here you put photo to svg - this is not any sence, you just put jpeg into svg. Open svg code of this image and you will see it.


Articles about form types
[1 article](https://www.pagecloud.com/blog/web-images-png-vs-jpg-vs-gif-vs-svg)
[2 article](https://webformyself.com/gif-png-jpg-ili-svg-chto-ispolzovat)


P.S. Please use jpeg here ^__^"